lectures/linear_equations.md

```diff
@@ -411,20 +389,15 @@ x @ y            # Another way to compute the inner product
 ```
 
 ```{code-cell} ipython3
-np.sqrt(np.sum(x**2))  # Norm of x, take one
+np.sqrt(np.sum(x**2))  # Norm of x, method one
 ```
 
 ```{code-cell} ipython3
-np.linalg.norm(x)      # Norm of x, take two
+np.linalg.norm(x)      # Norm of x, method two
 ```
 
-
-
 ## Matrix operations
 
-```{index} single: Matrix; Operations
```

Comment:
@HengchengZhang any reason we are removing the index entries that show up in the `pdf` version?

Reply:
Hi @mmcky I just find that it doesn't affect the current lectures and we have this only in some lectures. I was trying to make things consistent but it seems like we should add this to every lecture instead of removing them?

Reply:
these objects only get used in `pdf` production so that is the document to check as it builds an `index` object.